package.json

```diff
@@ -14,7 +14,7 @@
     "chai": "^4.0.1",
     "chai-as-promised": "^7.1.1",
     "dirty-chai": "^2.0.1",
-    "eslint": "^4.19.1",
+    "eslint": "^6.8.0",
```

Comment:
I don't like having to drop support for versions of Node in order to support a dev dependency. Prefer to get some usage numbers first to determine if it's okay to drop these first.

Reply:
Can we use a 5.X version for now and/or move this to another PR?

Reply:
Done: https://github.com/sendgrid/sendgrid-nodejs/pull/1063/files